packages/react-core/src/components/Nav/NavItem.tsx

```diff
@@ -67,9 +67,9 @@ export const NavItem: React.FunctionComponent<NavItemProps> = ({
   const { isSidebarOpen } = React.useContext(PageSidebarContext);
   const [flyoutTarget, setFlyoutTarget] = React.useState(null);
   const [isHovered, setIsHovered] = React.useState(false);
-  const ref = React.useRef<HTMLLIElement>();
+  const ref = React.useRef<HTMLLIElement>(undefined);
   const flyoutVisible = ref === flyoutRef;
-  const popperRef = React.useRef<HTMLDivElement>();
+  const popperRef = React.useRef<HTMLDivElement>(undefined);
   const hasFlyout = flyout !== undefined;
   const Component = hasFlyout ? 'button' : (component as any);
```

Comment:
similar to other comment e.g. using any type for elements... seems like we really shouldn't be introducing what essentially amounts to untyped components by using any here and many other places